standalone/nw.js

```diff
@@ -0,0 +1,74 @@
+function patchSqueakForNode(root) {
+    var fs = require("fs");
+    var Squeak = window.Squeak;
```

Comment:
Hi @bertfreudenberg node has some global objects:
https://nodejs.org/api/globals.html

But I think the best is use none like that and in each `.js`  file we use `.export` and `require` accordingly. Works perfectly well and solves the async load behind the scenes.

Take a look at my work in progress...

Here I'm importing `Squeak`https://github.com/sebastianconcept/SqueakJS/blob/feature/nodeJS-experiment/node/headless.js#L30

which got exported from here:
https://github.com/sebastianconcept/SqueakJS/blob/feature/nodeJS-experiment/node/vm.js#L7138

Have in mind that what I'm doing is very drafty to reach a proof of concept. I expect it to be heavily reviewed once I myself better in how the whole thing works.